examples/rust/extend_viewer_ui/src/main.rs

```diff
@@ -0,0 +1,198 @@
+//! This example shows how to wrap the Rerun Viewer in your own GUI.
+
+use re_viewer::external::{
+    arrow2, eframe, egui, re_arrow_store, re_data_store, re_log, re_log_types, re_memory, re_query,
+};
+
+// By using `re_memory::AccountingAllocator` Rerun can keep track of exactly how much memory it is using,
+// and prune the data store when it goes above a certain limit.
+// By using `mimalloc` we get faster allocations.
+#[global_allocator]
+static GLOBAL: re_memory::AccountingAllocator<mimalloc::MiMalloc> =
+    re_memory::AccountingAllocator::new(mimalloc::MiMalloc);
+
+#[tokio::main]
+async fn main() -> Result<(), Box<dyn std::error::Error>> {
+    // Direct `log` calls to stderr
+    re_log::setup_native_logging();
+
+    // Log crashes and signals, and send them to analytics if
+    // the `analytics` feature is on in `Cargo.toml`
+    re_crash_handler::install_crash_handlers(re_viewer::build_info());
+
+    // Listen for TCP connections from logging SDKs.
```

Comment:
Three different meanings of the term "log" in just a couple lines. Potentially confusing?